vendor/modules.txt

```diff
@@ -9,8 +9,16 @@ github.com/Microsoft/go-winio
 github.com/Microsoft/go-winio/pkg/guid
 # github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5
 github.com/Nvveen/Gotty
+# github.com/OneOfOne/xxhash v1.2.8
+github.com/OneOfOne/xxhash
```

Comment:
Where do we use this? Is this OPA dependency?

Reply:
yes